api/db/populate-events.js

```diff
@@ -76,7 +76,7 @@ const createEvents = async (multiplier, randomLocation, verbose) => {
       separator: ' ',
     });
 
-    const description = txtgen.article(Math.floor(Math.random() * 2) + 1);
+    const description = txtgen.article(Math.ceil(Math.random() * 2));
```

Comment:
While we are at it, we could change this `2` for a `3`, as this was another suggestion of mine from when I thought that `Math.random()` could return `1`